global.json

```diff
@@ -7,8 +7,9 @@
       ]
     },
     "vs-opt": {
-      "version": "16.8"
-    }
+      "version": "16.9"
+    },
+    "xcopy-msbuild": "16.8.0-preview3"
```

Comment:
Note that Jared manually releases new packages only when we ask him, so this will make it more difficult to test full framework changes.

Reply:
Yep, he is OOF, so I need to make the package myself. I found the instruction by asking around. I'll do that later